services/post.service.js

```diff
@@ -0,0 +1,23 @@
+import { EntityService } from './entity.service.js';
+
+export class PostService {
+  constructor(db) {
+    this.service = new EntityService(db, 'posts');
```

Comment:
Please, read about the proper naming of the tables in the database(?posts -> post, users -> user)